### PR TITLE
Update setting-up-cloud-native-observability-on-a-vm.md

### PR DESCRIPTION
**Pull request by samgnaniah**
Fixes https://github.com/wso2/docs-apim/issues/5838

### CODE REVIEW

File: en/docs/observe/micro-integrator/setting-up-cloud-native-observability-on-a-vm.md [hunk 1]
```diff
@@ -122,6 +122,9 @@ After applying the above change, you can start the Micro Integrator with the fol
 
 Once you have successfully set up the [minimum deployment](#step-1-set-up-the-minimum-deployment), you need to set up the log processing add-on to process logs. To achieve this, you can use Grafana Loki-based logging stack.
 
+!!! note
+    You do not need the minimum deployment to set this up and observe the traces through Jaeger. 
+
```
Comment (Rashmika998) [inline en/docs/observe/micro-integrator/setting-up-cloud-native-observability-on-a-vm.md:127]: This should be applied under Step 3 - Optionally, integrate the Message Tracing add-on.

Comment (Rashmika998) [inline en/docs/observe/micro-integrator/setting-up-cloud-native-observability-on-a-vm.md:127]: This should be applied under Step 3 - Optionally, integrate the Message Tracing add-on.

File: en/docs/observe/micro-integrator/setting-up-cloud-native-observability-on-a-vm.md [hunk 2]
```diff
@@ -348,24 +351,8 @@ Follow the instructions below to configure the Micro Integrator to publish traci
...
-    "opentracing.logs": "synapse_properties.'jaeger.reporter.log.spans'",
-    "opentracing.manager_host": "synapse_properties.'jaeger.sampler.manager.host'",
-    "opentracing.agent_host": "synapse_properties.'jaeger.sender.agent.host'"
-    ```
-!!! note
-    The service name used to initialize the JaegerTracer can be configured using the environment variable `SERVICE_NAME`
-    as shown below.
-    ```
-    export SERVICE_NAME=customServiceName
+    agent_host = "localhost"
```
Comment (Rashmika998) [inline en/docs/observe/micro-integrator/setting-up-cloud-native-observability-on-a-vm.md:354]: This change should be applied under Step 3.2 - Set up the Micro Integrator.

Comment (Rashmika998) [inline en/docs/observe/micro-integrator/setting-up-cloud-native-observability-on-a-vm.md:354]: This change should be applied under Step 3.2 - Set up the Micro Integrator.

File: en/docs/observe/micro-integrator/setting-up-cloud-native-observability-on-a-vm.md (hunk 1 above)
Comment (samgnaniah) [reply en/docs/observe/micro-integrator/setting-up-cloud-native-observability-on-a-vm.md]: Fixed.

Comment (samgnaniah) [reply en/docs/observe/micro-integrator/setting-up-cloud-native-observability-on-a-vm.md]: Fixed.

File: en/docs/observe/micro-integrator/setting-up-cloud-native-observability-on-a-vm.md [hunk 3]
```diff
@@ -348,24 +351,8 @@ Follow the instructions below to configure the Micro Integrator to publish traci
...
-    "opentracing.enable": "synapse_properties.'opentracing.enable'",
-    "opentracing.logs": "synapse_properties.'jaeger.reporter.log.spans'",
-    "opentracing.manager_host": "synapse_properties.'jaeger.sampler.manager.host'",
-    "opentracing.agent_host": "synapse_properties.'jaeger.sender.agent.host'"
-    ```
-!!! note
-    The service name used to initialize the JaegerTracer can be configured using the environment variable `SERVICE_NAME`
-    as shown below.
-    ```
-    export SERVICE_NAME=customServiceName
```
Comment (Rashmika998) [inline en/docs/observe/micro-integrator/setting-up-cloud-native-observability-on-a-vm.md:366]: Revert this.

**Review by Rashmika998 — APPROVED** [review]

**Review by Mariangela — APPROVED** [review]

**Pull request merged**